README.md

```diff
@@ -71,7 +82,7 @@ Do this for each client connecting to the server.
 
 	**Method #1 - Hosts**
```

Comment:
I have a question about certificate above. Is this step still required?
I am asking because everything works fine for me without this certificate :)

Reply:
now this is a surprise